src/components/icons/TokenIcon.tsx

```diff
@@ -31,6 +32,7 @@ export const TOKEN_ICONS: Record<string, string> = {
   DAI,
   BUSD,
   MTA,
+  SUSD,
```

Comment:
Should use `symbol.toUpper` in the code below and keep all assets uppercase here. SUSD is cap on Ropsten but sUSD on mainnet